features-json/css-container-queries.json

```
@@ -1,8 +1,8 @@
 {
   "title":"CSS Container Queries",
   "description":"Experimental media query to apply styles based on specified container elements rather than the entire page. Currently implemented using the `container` properties and `@container` at-rule.",
-  "spec":"https://drafts.csswg.org/css-contain-3/",
-  "status":"unoff",
+  "spec":"https://www.w3.org/TR/css-contain-3/",
+  "status":"WD",
```

Great! 🎉

Lowercase `wd` should make the checks happy :)

```suggestion
  "status":"wd",
```